data/mods/MindOverMatter/monsters/feral_psychics.json

```diff
@@ -1445,5 +1445,110 @@
       ],
       "flags": [ "NO_FUNG_DMG", "WIELDED_WEAPON" ]
     }
+  },
+  {
+    "id": "mon_feral_human_vita3",
+    "type": "MONSTER",
+    "name": "font of vitality",
+    "description": "This is the healthiest person you've even seen, especially after the end of the world.  Their skin positively glows, and as you watch them brush past a jagged chunk of wood to get to you, the wound they receive immediately stops bleeding and rapidly begins to close.",
```

Comment:
I don't think referring to an environmental component(jagged chunk of wood) you won't always find them in makes sense for indicating their regeneration.

Reply:
This is true but wood was the least bad way to make the point because it was the only material I could think of that you could reasonably find in an unspoiled wilderness, a suburban house, an industrial facility, and a secret lab. Metal and rock don't work, neither would rubble (rubble in the wilderness?) but wood is everywhere.